packages/angular/projects/ui-angular/src/lib/components/amplify-confirm-sign-up/amplify-confirm-sign-up.component.html

```diff
@@ -2,41 +2,36 @@
   Define default contents here
 -->
 <ng-template #confirmSignUpButton>
-  <button data-amplify-button type="submit">{{ confirmText }}</button>
+  <button amplify-button variation="primary" fullWidth="true" type="submit">
+    {{ confirmText }}
+  </button>
```

Comment:
I'm using directives to declare `amplify-button` primitive. That way it does not put a `amplify-button` element wrapper around the button, ie. we get `<button></button>` instead of `<amplify-button><button></button></amplify-button>`. 

This allows us to directly place attributes on the `button`, and ensures that css grids and flexbox style are applied correctly to the button and not to its wrapper.

This pattern is used for angular [bootstrap](https://ng-bootstrap.github.io/#/components/buttons/examples) and [material](https://material.angular.io/components/button/examples). I'll be using this pattern for primitives that consists of one html element (button, label, ...).

Reply:
This is only used internally for now and its spec can change later.

Reply:
Nice clever! Yeah I saw the selector ` selector: 'button[amplify-button]',` for you Amplify button component and that makes sense now. I think you can have that just `[amplify-button]` and it would work too not just buttons. But either way good job

Reply:
Yeah! That would be nice too, so that it can be applied to links and other elements as well. I'll take note 📝